src/main/java/com/sun/wineshop/controller/OrderController.java

```diff
@@ -31,4 +29,17 @@ public ResponseEntity<BaseApiResponse<OrderResponse>> placeOrder(@RequestBody Pl
                 messageUtil.getMessage("order.placed.success")
         ));
     }
+
+    @GetMapping("/{orderId}")
+    public ResponseEntity<BaseApiResponse<OrderDetailResponse>> getOrderDetail(
```

Comment:
```suggestion
    public ResponseEntity<BaseApiResponse<OrderDetailResponse>> show(
```
tên dễ thương của nó là Show đó các bạn 
nếu thích đổi, ko thích thì thôi